src/Sylius/Bundle/TaxonomyBundle/Doctrine/ORM/TaxonRepository.php

```diff
@@ -116,6 +116,7 @@ public function findOneByName($name)
             ->leftJoin('o.translations', 'translation')
             ->where('translation.name = :name')
             ->setParameter('name', $name)
+            ->setMaxResults(1)
```

Comment:
So the issue is, if an entity that's translatable has 2 items with the same name, getOneOrNullResult will actually return an exception. If you read the linked issue it makes more sense